cluster/ubuntu/util.sh

```diff
@@ -107,7 +109,7 @@ function trap-add {
   trap "${handler}" ${signal}
```

Comment:
I'm trying to use single quotes here, yet it complains that `handler` is a binding var. 
I'll dive down later.